examples/td3/config.yaml

```diff
@@ -1,47 +1,49 @@
-# Environment
+# task and env
 env:
   name: HalfCheetah-v3
   task: ""
-  exp_name: "HalfCheetah-TD3"
+  exp_name: "HalfCheetah-TD3-ICLR"
```

Comment:
Can we avoid any specific reference to a submission that we may forget to remove?